### PR TITLE
fix permissions for fpm configuration

### PR DESCRIPTION
**Pull request by pkubatrh**

### CODE REVIEW

**Comment by pkubatrh** [comment]
Follow up for #401 as I missed some fpm configuration in 8.0 and 8.1

**Comment by phracek** [comment]
The fix is part of #406. Only for info.

**Review by phracek — CHANGES_REQUESTED** [review]
Everything LGTM. What about adding as well www.conf file like:
`chmod a+rwx ${PHP_FPM_CONF_D_PATH}/${PHP_FPM_CONF_FILE}`
${PHP_SYSCONF_PATH}/php-fpm.d equals to ${PHP_FPM_CONF_D_PATH}.

Can you please modify it?

**Comment by pkubatrh** [comment]
> What about adding as well [www.conf](http://www.conf/) file like: chmod a+rwx ${PHP_FPM_CONF_D_PATH}/${PHP_FPM_CONF_FILE}

The chmod call is recursive, so that should not be needed. I have modified the original call to use `${PHP_FPM_CONF_D_PATH}` however.

**Comment by pkubatrh** [comment]
[test]

**Review by phracek — CHANGES_REQUESTED** [review]
Please address my changes.

File: 8.0/root/usr/libexec/container-setup [hunk 1]
```diff
@@ -51,6 +51,7 @@ if [ "$PLATFORM" == "el7" ]; then
 else
   chmod a+rwx "${PHP_SYSCONF_PATH}/php.ini"
   chmod -R a+rwx "${PHP_SYSCONF_PATH}/php.d"
+  chmod -R a+rwx "${PHP_SYSCONF_PATH}/php-fpm.d"
 fi
 
 if [ "x$PLATFORM" == "xel9" ] || [ "x$PLATFORM" == "xfedora" ]; then
```
Comment (phracek) [inline 8.0/root/usr/libexec/container-setup:57]:
```suggestion
if [ "x$PLATFORM" == "xel9" ] || [ "x$PLATFORM" == "xfedora" ]; then
chmod -R a+rwx "${PHP_SYSCONF_PATH}/php-fpm.d"
```

**Comment by phracek** [comment]
[test]

**Review by phracek — APPROVED** [review]
LGTM. Thanks for the fix

**Pull request merged**